charts/camunda-platform/templates/zeebe/statefulset.yaml

```diff
@@ -85,6 +85,35 @@ spec:
               value: {{ .Values.zeebe.ioThreadCount  | quote }}
             - name: ZEEBE_BROKER_GATEWAY_ENABLE
               value: "false"
+            {{- if .Values.global.elasticsearch.external  }}
+              {{- if .Values.global.elasticsearch.openSearch}}
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_AUTHENTICATION_USERNAME
+              value: {{ .Values.global.elasticsearch.auth.username | quote }}
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_AUTHENTICATION_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: elastic-password
+                  key: password
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_CLASSNAME
+              value: "io.camunda.zeebe.exporter.opensearch.OpensearchExporter"
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_URL
+              value: {{ include "camundaPlatform.elasticsearchURL" . | quote }}
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_BULK_SIZE
+              value: "1"
```

Comment:
❌ why do we set this to 1, the default within zeebe is `1000`? I would be in favor of keeping the app default and optionally allow the user to override it

Reply:
@megglos 
I was following the env vars mentioned in the official docs: https://docs.camunda.io/docs/next/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/irsa/#zeebe
As you can see, the bulk size is set to 1 in the docs. I  can remove that env var and test zeebe again.

Reply:
removed `ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_BULK_SIZE`

Reply:
I think it is just an example, [here](https://docs.camunda.io/docs/next/self-managed/zeebe-deployment/exporters/elasticsearch-exporter/?configuration=bulk) is mentioned that the default value is 1000